src/test/java/org/jabref/logic/openoffice/oocsltext/CSLFormatUtilsTest.java

```diff
@@ -0,0 +1,721 @@
+                ),
+
+                // Type: "1."
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "The Journal of Veterinary Medical Science".equals(e.getTitle())).findAny().orElse(null),
+                        "  \n" +
+                                "    3. Smith, B., Jones, B. and Williams, J. 2016. Title of the test entry. <i>BibTeX Journal</i>. <b>34</b>: 45–67.\n" +
+                                "  \n"
+                ),
+
+                // Type:"<BOLD>1."
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "Acta Orthopædica Belgica".equals(e.getTitle())).findAny().orElse(null),
+                        "  \n" +
+                                "    <b>3</b>. <b>Smith  B, Jones  B, Williams  J</b>. Title of the test entry. <i>BibTeX Journal</i> 2016 ; 34 : 45–67.\n" +
+                                "  \n"
+                ),
+
+                // Type: "1"
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "Acta Anaesthesiologica Taiwanica".equals(e.getTitle())).findAny().orElse(null),
+                        "  \n" +
+                                "    3 Smith Bill, Jones Bob, Williams Jeff. Title of the test entry. <i>BibTeX Journal</i> 2016;<b>34</b>(3):45–67. Doi: 10.1001/bla.blubb.\n" +
+                                "  \n"
+                ),
+
+                // Type: "(1)"
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "American Chemical Society".equals(e.getTitle())).findAny().orElse(null),
+                        "  \n" +
+                                "    (3) Smith, B.; Jones, B.; Williams, J. Title of the Test Entry. <i>BibTeX Journal</i> <b>2016</b>, <i>34</i> (3), 45–67. https://doi.org/10.1001/bla.blubb.\n" +
+                                "  \n"
+                ),
+
+                // Type: "1)"
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "Chemical and Pharmaceutical Bulletin".equals(e.getTitle())).findAny().orElse(null),
+                        "  \n" +
+                                "    3) Smith B., Jones B., Williams J., <i>BibTeX Journal</i>, <b>34</b>, 45–67 (2016).\n" +
+                                "  \n"
+                ),
+
+                // Type: "<SUPERSCRIPT>1"
+                Arguments.of(
+                        STYLE_LIST.stream().filter(e -> "American Institute of Physics 4th edition".equals(e.getTitle())).findAny().orElse(null),
+                        "  <sup>3</sup> B. Smith, B. Jones, and J. Williams, “Title of the test entry,” BibTeX Journal <b>34</b>(3), 45–67 (2016).\n"
+                )
+        );
+    }
+
+    /**
+     * Tests if a citation (LaTeX "\cite") is converted into an in-text citation (LaTeX "\citet") as expected.
+     */
+    @ParameterizedTest
+    @MethodSource("provideCitations")
+    void testChangeToInText(String input, String expected) {
```

Comment:
OpenRewrite should complain here -> no `test` prefix in method names.

https://docs.openrewrite.org/recipes/java/testing/cleanup/removetestprefix

Activated at https://github.com/JabRef/jabref/blob/23727b7ae1ff9937c02288c069451ea9ad215ba3/rewrite.yml#L199 and https://github.com/JabRef/jabref/blob/23727b7ae1ff9937c02288c069451ea9ad215ba3/build.gradle#L368.

`main` is also up to date (via 8a23b9361c666a20331)

Reply:
- Fixed in https://github.com/openrewrite/rewrite-testing-frameworks/commit/8e001a92c6c9097494db3ac21d70474dcfd2f7ca

Reply:
Method name fixed.